.github/workflows/versioned-report.yml

```diff
@@ -26,6 +28,8 @@ jobs:
 
       - name: List all Bowtie Supported Implementations having a "matrix-versions.json" file
         id: implementations-matrix
+        env:
+          GH_TOKEN: ${{ github.token }} # for GH API access
```

Comment:
Remind me, this is because in the script we have our "own" env var which is used for building the package so that we don't overlap with the `GITHUB_TOKEN` used for `github3.py` right?

Can you add a comment just reminding that here?

Reply:
No, this is because the [GH CLI](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/using-github-cli-in-workflows) requires that variable to have access to the github token